pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -79,8 +79,8 @@ func (r *REST) NewList() runtime.Object {
 var _ = rest.Creater(&REST{})
 
 var (
-	forbiddenNames    = []string{"openshift", "kubernetes", "kube"}
-	forbiddenPrefixes = []string{"openshift-", "kubernetes-", "kube-"}
+	ForbiddenNames    = []string{"openshift", "kubernetes", "kube"}
```

Comment:
Kubernetes and Kube aren't reserved namespaces, remove from this list (can be followup)

Reply:
We reserved them from being requested since we reserve the kube and kubernetes prefixes